dask/dataframe/io/csv.py

```diff
@@ -24,30 +24,33 @@
 delayed = delayed(pure=True)
 
 
-def bytes_read_csv(b, header, kwargs, dtypes=None, columns=None,
-                   write_header=True, enforce=False):
+def bytes_read_pandas(reader, b, header, kwargs, dtypes=None, columns=None,
+                      write_header=True, enforce=False):
```

Comment:
This could use a more specific name.  What kinds of bytes are these?  They could be, for example bytes from HDF5.

This function name comes up in the progress bars, and so is very user facing.